themes/custom/az_barrio/css/uaqs_navigation_overlay_menu_scroll.css

```diff
@@ -1,3 +1,8 @@
+/* For structuring header utilities*/
+header#header_arizona {
+  min-height: 50px;
+}
+
```

Comment:
Looks like this `uaqs_navigation_overlay_menu_scroll.css` file isn't associated with any of AZ Barrio's [libraries](https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/az_barrio.libraries.yml) and isn't really used anywhere...

Reply:
Could this be related to some previous/current issues of Quickstart not using Bootstrap styles? Let's do the 2 ticket solution like we discussed this morning for this one.